doozerlib/cli/release_gen_payload.py

```diff
@@ -114,11 +114,13 @@ def modify_and_replace_api_object(api_obj: oc.APIObject, modifier_func: Callable
               help="If doozer should also create a release payload for multi-arch/heterogeneous clusters.")
 @click.option("--moist-run", default=False, is_flag=True,
               help="Performing mirroring/etc but to not actually update imagestreams.")
+@click.option("--publish", default=False, is_flag=True,
```

Comment:
Parameter name `--publish` sounds misleading.

Reply:
this parameter is inherit from jenkins job's parameter

Reply:
But this is not a jenkins job. Jenkins job assumes some certain use cases but our Doozer may need more flexibility. 
An option name like `--also-publish-to=namespace` or split the publishing step into separate Doozer verb / artcd step could be a better idea.

Reply:
realistically i don't think there's a use for this doozer verb outside a job like build-sync. so inlining job logic makes sense.

what's misleading about `--publish`? it creates a release image where people can get to it to test it. registry.ci isn't exactly public but otherwise... that's publishing :) however the param could probably benefit from a more explicit name, to distinguish from what the `promote` job does. maybe `--publish-test-image`?